textproc/ocrmypdf/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                ocrmypdf
+version             16.1.2
+revision            0
+categories          textproc
+
+homepage            https://github.com/ocrmypdf/OCRmyPDF
+
+maintainers         {@akierig fastmail.de:akierig} openmaintainer
+
+
+checksums           rmd160  25b94fbd304d1fb8235cbd12c1b8fb2af0cf868a \
+                    sha256  d1f9c62b3a0cb090218e8f719d85d110bad95cb73e83dec05842652d173b0c29 \
+                    size    6720680
+
+description         ${name} adds an OCR text layer to scanned PDF files, \
+                    allowing them to be searched
+long_description    ${description}. ${name} also supports plugins \
+                    that enable customization of its processing steps, \
+                    and it is highly tolerant of PDFs containing scanned \
+                    images and “born digital” content that doesn’t require text recognition.
+
+supported_archs     noarch
+platforms           {darwin any}
+license             MPL-2
+python.default_version 312
+
+depends_build-append port:py${python.version}-setuptools_scm
```

Comment:
this needs to be aligned with spaces in multiples of four. Typically, one would add a line-continuation and then have the dependency on the next line aligned correctly.

Reply:
Ahh. I did wonder about that. in any case, fixed.